zds/tutorialv2/tests/tests_views/tests_published.py

```diff
@@ -1988,7 +1988,7 @@ def test_author_update(self):
     def check_images_socials(self, result, pattern_link_image, title, description):
         start_reg = r"<meta(\s+)(\S+)"
 
-        for meta in ["twitter:image", "og:image:url"]:
+        for meta in ["X:image", "og:image:url"]:
```

Comment:
```suggestion
        for meta in ["twitter:image", "og:image:url"]:
```

Le nom des paramètres `meta` [n’a pas changé](https://developer.x.com/en/docs/twitter-for-websites/cards/overview/markup).